Sources/SBTUITestTunnelClient/include/SBTUITestTunnelClientProtocol.h

```diff
@@ -29,6 +29,13 @@
  */
 - (void)launchTunnel;
 
+/**
+ *  Launch application synchronously waiting for the tunnel server connection to be established.
```

Comment:
```suggestion
 *  Launches the application synchronously using the provided URL waiting for 
 *  the tunnel server connection to be established.
```

The same applies to all other methods specifying url